client/src/main/java/com/arcbees/beeshop/client/application/home/HomeView.java

```diff
@@ -26,8 +30,28 @@
     interface Binder extends UiBinder<Widget, HomeView> {
     }
 
+    @UiField(provided = true)
+    PricePresenter shirtPrice;
+    @UiField(provided = true)
+    PricePresenter bagPrice;
+
     @Inject
-    HomeView(Binder uiBinder) {
+    HomeView(
+            Binder uiBinder,
+            PriceWidgetFactory priceWidgetFactory) {
+        ProductDto shirt = new ProductDto();
+        shirt.setBrandName("Bee");
+        shirt.setPrice(55);
+        shirt.setProductName("Shirt");
+
+        ProductDto bag = new ProductDto();
+        bag.setBrandName("Bee");
+        bag.setPrice(35);
+        bag.setProductName("Bag");
+
+        shirtPrice = priceWidgetFactory.create(shirt);
```

Comment:
Presenters should be created by other presenters and added to the view through slots. Otherwise don't use a presenter here.